src/main/java/ee/buerokratt/ruuter/controller/DslController.java

```diff
@@ -20,25 +20,25 @@
 @Slf4j
 @RestController
 @RequiredArgsConstructor
-public class ConfigurationController {
-    private final ConfigurationService configurationService;
+public class DslController {
+    private final DslService dslService;
     private final ApplicationProperties properties;
 
-    @RequestMapping(value = "/{configuration}")
-    public ResponseEntity<RuuterResponse> queryConfiguration(@PathVariable String configuration,
-                                                             @RequestBody(required = false) Map<String, Object> requestBody,
-                                                             @RequestParam(required = false) Map<String, Object> requestParams,
-                                                             HttpServletRequest request) {
+    @RequestMapping(value = "/{dsl}")
```

Comment:
Update the GUIDE as well, to reference the new `/{dsl}` instead of the old `/configuration`